### ISSUE TITLE
Add a method on ValueUserOverride to clear all overrides

### ISSUE THREAD

**Issue by XDelta** [Yellow-Dog-Man/Resonite-Issues#974]
### Is your feature request related to a problem? Please describe.

I'd like to be able to clear/reset all overrides on a `ValueUserOverride` from Protoflux. In a game world I work on, there are many localized values using `ValueUserOverride` and in several cases it would be useful to clear them all.

### Describe the solution you'd like

I'd like to be able to use the MethodProxy node to call this method from protoflux to easily remove all entries.

### Describe alternatives you've considered

Make each user locally write the target setting back to the default.

**Comment by JackTheFoxOtter** [comment]
I would additionally like to see an action to clear the override for the local user only, to remove a specific (currently present) user.

**Comment by XDelta** [comment]
It unfortunately is about the same. although could be useful for more complex override types. Being able to fully just clear the list in this case allows resetting values for users who aren't in the session at that exact time but may return later

**Comment by Zyzyl** [comment]
> Based on your description however- it sounds like broadly- it'd just be more useful if the component had another toggle like the persistence toggle- but acts on user join/leave instead of per-session- it'd enable a setup like described above- but would be much easier to set up as it would be built-in to the component. I could see use in that.

Agreed that this would be useful - already requested here: https://github.com/Yellow-Dog-Man/Resonite-Issues/issues/1013

**Comment by JackTheFoxOtter** [comment]
> It allows you to call the "Set Active" method which can seemingly be used to locally overwrite a user's VUO value to whatever the default is. It won't remove their entry from the list- but it will allow you to set their defaults back.

This only works for users that are currently in the session though. There is currently no way to reset all overwrites of a ValueUserOverwrite unless all users with a value are currently present. That's the big limitation a "clear all" option would remove.

**Comment by JackTheFoxOtter** [comment]
> Based on your description however- it sounds like broadly- it'd just be more useful if the component had another toggle like the persistence toggle- but acts on user join/leave instead of per-session- it'd enable a setup like described above- but would be much easier to set up as it would be built-in to the component. I could see use in that.

Yeah, as Zyzyl mentioned I already requested that in #1013, however, I think both of these are useful in their own right. Sometimes you want to be able to only clear values when a specific event happens, but would generally want to keep values persistent. Sometimes you want to make it automatically clear the overwrite when the user leaves.

**Comment by JackTheFoxOtter** [comment]
Now that #1014 is implemented, this also applies to BooleanUserOverrideGather and NumericUserOverrideGather components.
